docs/kyma/docs/040-cr-installation.md

```diff
@@ -12,7 +12,7 @@ kubectl get crd installations.installer.kyma-project.io -o yaml
 
 ## Sample Custom Resource
 
-This is a sample CR that controls the Kyma installer. This example has the **action** label set to `install`, which means that it triggers the installation of Kyma.
+This is a sample CR that controls the Kyma installer. This example has the **action** label set to `install`, which means that it triggers the installation of Kyma. The  **name** and **namespace**  fields in the `components` array define which components you install and Namespaces in which you install them. This example shows that you install all components except for `remote-environments`. If you want the release name to be different from the component's name, provide the release parameter.
```

Comment:
This is dangerous: "install all components except for `remote-environments`"
Please describe the convention used instead. It is the folder name in the resources folder from the URL or injected sources.

Reply:
i guess that's already explained in the table. But would it be better to come back to previous example:
```
    - name: "remote-environments"
      namespace: "kyma-integration"
      release: "hmc-default"
```